elasticsearch-core/src/test/scala/com/sumologic/elasticsearch/restlastic/RestlasticSearchClientTest.scala

```diff
@@ -168,6 +168,52 @@ class RestlasticSearchClientTest extends WordSpec with Matchers with ScalaFuture
         res.jsonStr should not include("doc5")
       }
     }
+    
+    "Support bulk deletion" in {
```

Comment:
Either add deletion to the previous test, or do the setup without checking the result to avoid duplicate tests.